packages/@react-types/button/src/index.d.ts

```diff
@@ -65,10 +65,19 @@ interface AriaBaseButtonProps extends FocusableDOMProps, AriaLabelingProps {
 export interface AriaButtonProps<T extends ElementType = 'button'> extends ButtonProps, LinkButtonProps<T>, AriaBaseButtonProps {}
 export interface AriaToggleButtonProps<T extends ElementType = 'button'> extends ToggleButtonProps, AriaBaseButtonProps, AriaButtonElementTypeProps<T> {}
 
+/** @deprecated */
+type LegacyButtonVariant = 'cta' | 'overBackground';
 export interface SpectrumButtonProps<T extends ElementType = 'button'> extends AriaBaseButtonProps, ButtonProps, LinkButtonProps<T>, StyleProps {
   /** The [visual style](https://spectrum.adobe.com/page/button/#Options) of the button. */
-  variant: 'cta' | 'overBackground' | 'primary' | 'secondary' | 'negative',
-  /** Whether the button should be displayed with a quiet style. */
+  variant: 'accent' | 'primary' | 'secondary' | 'negative' | LegacyButtonVariant,
+  /** The background style of the button. */
+  style?: 'fill' | 'outline',
```

Comment:
this one needs a `@default 'fill'`?

Reply:
The default depends on the variant...

Reply:
o, haha.... was looking at the spectrum design props table